election-api-java/src/main/java/bbc/news/elections/model/ApiResponse.java

```diff
@@ -1,11 +1,11 @@
 package bbc.news.elections.model;
 
-import lombok.Builder;
-import lombok.Value;
-
-@Value
-@Builder
 public class ApiResponse {
-    String error;
-    String message;
+    final String error;
+    final String message;
```

Comment:
I'd expect the model classes to be more "bean-like" (private fields, public getters). If the intention is to forego encapsulation then should these fields be public rather than package-private?

Reply:
That's a good idea, it's more idiomatic.  I'll do that (too used to Scala case classes).